### ISSUE TITLE
limit of the number of files in a single bucket

### ISSUE THREAD

**Issue by MeijeSibbel** [GooBox/goobox-sync-storj#16]
Currently if one uploads more than 1-2k files to a bucket, file listing which is necessary in order to upload/download or track changes will slow down to a halt. Though this is mostly a Storj issue, it is something we should be very careful about to prevent the complete app to slow down to a halt. There is also a hard limit to the number of files in a Storj bucket.

**Comment by kaloyan-raev** [comment]
I'd say we should first investigate the issue at Storj and understand what causes it and what the actual limits are. Working the issue around might not be a better solution at all.

**Comment by MeijeSibbel** [comment]
These are good issues to track on this: 

https://github.com/Storj/bridge/issues/534
https://github.com/Storj/libstorj/issues/363
https://github.com/Storj/bridge/issues/528

**Comment by MeijeSibbel** [comment]
I agree, for now we should just wait until Storj fixes this, a workaround is too much work right now.
